scripts/gather-files.js

```diff
@@ -8,11 +8,10 @@ const path = require('path');
  * @param {string} dir - Path to directory of files.
  * @return {array} – array of file contents
  */
-
+const hiddenFileRegex = /(^|\/)\.[^/.]/g;
 function gatherFiles(dir) {
-  return fs
-    .readdirSync(dir)
-    .map(f => fs.readFileSync(path.join(dir, f), 'utf8'));
+  const list = fs.readdirSync(dir).filter(item => !hiddenFileRegex.test(item));
```

Comment:
Oh nice catch. Couldn't this be much simpler and exclude anything that isn't .svg  i.e `.filter(item => item.match(/.svg$/))`?